web_src/css/helpers.css

```diff
@@ -22,6 +22,7 @@ Gitea's private styles use `g-` prefix.
 .gt-vm { vertical-align: middle !important; }
 .gt-w-100 { width: 100% !important; }
 .gt-h-100 { height: 100% !important; }
+.gt-wh-100 {  width: 100% !important;  height: 100% !important; }
```

Comment:
Do we really have to do this? I generally prefer single-rule classes where practical.

Reply:
Oh, thought I went through the diff carefully to not include anything that shouldn't be committed.
That's an artifact from an alternative design that isn't used anymore in this PR.